ci-operator/step-registry/cucushift/upgrade/rhel-worker/cucushift-upgrade-rhel-worker-commands.sh

```diff
@@ -212,6 +212,24 @@ function rhel_post_upgrade(){
         echo "oc logs checking command failed." && return 1
     fi
 
+    cat > /tmp/post_check.yaml <<-'EOF'
+---
+- name: Run post check on the workers
+  hosts: workers
+  gather_facts: false
+  tasks:
+  - name: Ensure fixfiles_exclude_dirs contains '/var/lib/kubelet'
+    command: grep -q '/var/lib/kubelet' /etc/selinux/fixfiles_exclude_dirs
```

Comment:
The check can be applied to all supported versions ? Where is `/etc/selinux/fixfiles_exclude_dirs` installed from ? Is there some case which some systems does not have the file ?

Reply:
Ah good point, it's firstly added in the 4.14 playbook in https://github.com/openshift/openshift-ansible/blob/release-4.14/roles/openshift_node/tasks/selinux.yml, so added a version check here. 
After this version, if the file is not present at all, then it means this step was not executed.